code/modules/shuttle/shuttles/ert.dm

```diff
@@ -0,0 +1,110 @@
+/*
+	ERT Shuttles
+*/
+
+// Base ERT Shuttle
+/obj/docking_port/mobile/emergency_response
+	name = "ERT Shuttle"
+	ignitionTime = DROPSHIP_WARMUP_TIME
+	area_type = /area/shuttle/ert
+	width = 7
+	height = 13
+	var/list/doors = list()
+
+/obj/docking_port/mobile/emergency_response/Initialize(mapload)
+	. = ..()
+	for(var/place in shuttle_areas)
+		for(var/obj/structure/machinery/door/airlock/air in place)
+			doors += list(air)
+
+/obj/docking_port/mobile/emergency_response/enterTransit()
+	..()
+	control_doors("close")
+	control_doors("lock")
```

Comment:
![image](https://user-images.githubusercontent.com/66756236/208639962-60867865-1990-4e4b-9a42-4daeafc56a71.png)
unlock doors first or you can do funny things by opening then locking the doors open

Reply:
These issues should be resolved